src/index.ts

```diff
@@ -35,6 +35,7 @@ export type {
   ActorPartsFragment,
   AttachmentPartsFragment,
   CustomerActorPartsFragment,
+  CustomerGroupPartsFragment,
```

Comment:
I'd expose the `PageInfoParts` as well.

Reply:
added ty

Reply:
Actually `CustomersSort` and `CustomersFilter` too!